src/test/java/api/requests/RequestsAPIUpdatingTests.java

```diff
@@ -682,4 +682,49 @@ public void replacedRequestShouldOnlyIncludeStoredPropertiesInStorage()
     assertThat("patron group information should not be stored for proxying user",
       proxySummary.containsKey("patronGroup"), is(false));
   }
+
+  @Test
+  public void cannotReplaceRequestWithAnInactiveUser()
```

Comment:
I have just thought of a question around this. 

Cancelling a request is currently done by replacing it's representation. If a user is de-actived after a request is created, how does that request get addressed? 

Should it be possible to cancel a request for an inactive user?

Reply:
Could you look at these questions @mreno-EBSCO

Reply:
I've followed up and asked Cate this question